internal/store/live_migration.go

```diff
@@ -20,27 +20,28 @@ import (
 
 	"github.com/mongodb/mongodb-atlas-cli/internal/config"
 	atlas "go.mongodb.org/atlas/mongodbatlas"
+	atlasv2 "go.mongodb.org/atlas/mongodbatlasv2"
 )
 
 //go:generate mockgen -destination=../mocks/mock_live_migration_validations.go -package=mocks github.com/mongodb/mongodb-atlas-cli/internal/store LiveMigrationValidationsCreator,LiveMigrationCutoverCreator,LiveMigrationValidationsDescriber
 
 type LiveMigrationValidationsCreator interface {
-	CreateValidation(string, *atlas.LiveMigration) (*atlas.Validation, error)
+	CreateValidation(string, *atlasv2.LiveMigrationRequest) (*atlasv2.Validation, error)
```

Comment:
This PR should be the gold standard for migrations as we no longer have any v1 objects in place.